olive/model/hf_utils.py

```diff
@@ -370,6 +369,15 @@ def get_hf_model_dummy_input(model_name: str, task: str, feature: Optional[str]
     return model_config.generate_dummy_inputs(tokenizer, framework="pt")
 
 
+def get_peft_task_type_from_task(task: str, fail_on_not_found=False) -> str:
+    """Get peft task type from feature"""
+    feature = TASK_TO_FEATURE.get(task, None)
+    peft_task_type = FEATURE_TO_PEFT_TASK_TYPE.get(feature, None) if feature else None
+    if peft_task_type is None and fail_on_not_found:
```

Comment:
>fail_on_not_found [](http://example.com/codeflow?start=34&length=17)

for the else case, do we need add logging?

Reply:
yeah, I can add a warning log for the else case

Reply:
Done!